ui/js/src/kimchi.api.js

```diff
@@ -372,22 +372,38 @@ var kimchi = {
         }).done(function(data, textStatus, xhr) {
             url = 'https://' + location.hostname + ':' + proxy_port;
             url += server_root;
-            url += "/plugins/kimchi/spice_auto.html";
-            /*
-             * When using server_root we need pass the value with port
-             * argument to be used correctly by spice_auto.html scripts
-             */
-            url += "?port=" + proxy_port + server_root;
-            url += "&listen=" + location.hostname;
-            /*
-             * From python documentation base64.urlsafe_b64encode(s)
-             * substitutes - instead of + and _ instead of / in the
-             * standard Base64 alphabet, BUT the result can still
-             * contain = which is not safe in a URL query component.
-             * So remove it when needed as base64 can work well without it.
-             * */
-            url += "&token=" + wok.urlSafeB64Encode(vm).replace(/=*$/g, "");
-            url += '&encrypt=1';
+
+            let spice_html5 = true;
```

Comment:
@alinefm how to best get it from server?

Reply:
I am thinking on add it as a parameter to wok.conf file and read from there. What do you think?

Reply:
It's probably already there (I added __with_spice_web_client__ similar to __with_spice__ in config.py.in), but not sure how to best tell front-end about whether __with_spice__ or __with_spice_web_client__ is on

Reply:
Query `/plugins/kimchi/spice_auto.html` and `/plugins/kimchi/spice-web-client/index.html` and take one that does not give 404?